src/test/lib/parts_test.ts

```diff
@@ -431,4 +431,124 @@ suite('Parts', () => {
       });
     });
   });
+
+  suite('EventPart', () => {
```

Comment:
Want to add an integration test that uses `@event="{handleEvent: onSomething, capture: true}"` syntax in the template?

Reply:
yeah

Reply:
done

Reply:
Did you forget to push your changes?  Still only 1 commit on the PR...

Reply:
oops, indeed. pushed